### ISSUE TITLE
Allow top-level return

### ISSUE THREAD

**Issue by Ginden** [microsoft/TypeScript#45207]
# Suggestion

## 🔍 Search Terms

top-level return 

## ✅ Viability Checklist

My suggestion meets these guidelines:

* [x] This wouldn't be a breaking change in existing TypeScript/JavaScript code
* [x] This wouldn't change the runtime behavior of existing JavaScript code
* [x] This could be implemented without emitting different JS based on the types of the expressions
* [x] This isn't a runtime feature (e.g. library functionality, non-ECMAScript syntax with JavaScript output, new syntax sugar for JS, etc.)
* [x] This feature would agree with the rest of [TypeScript's Design Goals](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Design-Goals).

## ⭐ Suggestion

I would like to have option to use top-level return.

This is required feature to compile code for platforms like [plv8](https://plv8.github.io/#scalar-function-calls)

See  microsoft/monaco-editor#1069

This is likely to be not compatible with ES modules, but it's compatible with commonjs module system.

## 📃 Motivating Example

Basically, some enviroments wrap code in invisible function.

Eg. plv8 code

```sql
CREATE FUNCTION plv8_test(keys TEXT[], vals TEXT[]) RETURNS JSON AS $$
    return  {keys, vals};
$$ LANGUAGE plv8 IMMUTABLE STRICT;
```

Is underhood:

```js
(function(keys, vals) {
    return  {keys, vals};
})
```

## 💻 Use Cases

Example use case: plv8

As workaround I planned to use TypeScript plugin
